tests/datasets/api_tests.py

```diff
@@ -1086,7 +1100,7 @@ def test_export_dataset_gamma(self):
 
         self.login(username="gamma")
         rv = self.client.get(uri)
-        assert rv.status_code == 401
+        assert rv.status_code == 404
```

Comment:
@betodealmeida export and import datasets will not need a separate permission now. import needs `can_write` on `Dataset` and export `can_read` on `Dataset`